charts/rancher-logging-crd/105.1.0-rc.1+up4.10.0/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+annotations:
+  catalog.cattle.io/certified: rancher
```

Comment:
looks like we're missing a log of the chart annotations I would expect to see here, see [105.0.0+up4.8.0](https://github.com/rancher/charts/blob/dev-v2.10/charts/rancher-logging/105.0.0%2Bup4.8.0/Chart.yaml#L2-L14) for example.

Looks like these are still included in `packages/rancher-logging/generated-changes/patch/Chart.yaml.patch` so not sure why these aren't being included here.

Reply:
Are you sure these annotations should be included for the crd chart? Looking [here](https://github.com/rancher/charts/blob/dev-v2.10/charts/rancher-logging-crd/105.0.0%2Bup4.8.0/Chart.yaml) doesn't look like they are. Let me know if I'm missing something ;)

Reply:
Ope my mistake, thought I was looking at the main chart, this is good as it is